data/json/monsters/rodentkin.json

```
@@ -141,6 +141,45 @@
     "flags": [ "SEES", "SMELLS", "HEARS", "WARM", "SWIMS", "PATH_AVOID_DANGER_1", "CAN_DIG" ],
     "dissect": "dissect_rat_sample_small"
   },
+  {
+    "id": "mon_pack_rat",
+    "type": "MONSTER",
+    "name": { "str": "pack rat" },
+    "description": "This humanoid rodent stands over five feet tall, it pokes and bites at anything shiny, hastily shoving anything of interest into its mouth",
+    "default_faction": "ratkin",
+    "copy-from": "mon_big_rat",
+    "volume": "20 L",
+    "weight": "30 kg",
+    "hp": 60,
+    "speed": 90,
+    "attack_cost": 80,
+    "symbol": "R",
+    "color": "brown_green",
+    "melee_dice": 4,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "cut", "amount": 5 } ],
+    "dodge": 3,
+    "armor_bash": 5,
+    "armor_cut": 6,
+    "armor_bullet": 2,
+    "anger_triggers": [ "PLAYER_WEAK", "PLAYER_CLOSE", "FRIEND_ATTACKED", "STALK" ],
+    "path_settings": { "max_dist": 50, "avoid_traps": true, "avoid_sharp": true },
+    "death_drops": "ratkin_death_drops_pack-rat",
```

```suggestion
    "death_drops": "ratkin_death_drops_pack_rat",
```
Maybe name the id `ratkin_death_drops_pack_rat`